python/python_direct/ops.cpp

```diff
@@ -49,6 +48,454 @@ Returns
 -------
 Val or TensorView
     The sum of the inputs.
+)")
+      NVFUSER_DIRECT_BINDING_BINARY_OP(
+          "atan2",
+          atan2,
+          R"(
+Element-wise arctangent of lhs/rhs choosing the quadrant.
+
+Parameters
+----------
+lhs : Val or TensorView
+    The y-coordinates.
+rhs : Val or TensorView
+    The x-coordinates.
+
+Returns
+-------
+Val or TensorView
+    The angles in radians between the positive x-axis and the rays to the points (x,y).
+)")
+          NVFUSER_DIRECT_BINDING_BINARY_OP(
+              "div",
+              div,
+              R"(
+Element-wise integer division.
```

Comment:
This is not an integer division.

Reply:
> // div: don't promote to float, instead, truncate the result, this has the same
> // semantics as the C++'s operator /

from `csrc/ops/arith.h`

Reply:
Thanks a lot for double checking that. I think it's only saying that it doesn't `promote` the math. But it's not converting a floating point to an integer for truncating and it would still perform floating point division.